src/overlays/actors/ovl_Obj_Ending/z_obj_ending.c

```diff
@@ -20,34 +19,30 @@ const ActorInit Obj_Ending_InitVars = {
     (ActorFunc)ObjEnding_Update,
     (ActorFunc)ObjEnding_Draw,
 };
-*/
 
-extern InitChainEntry D_80C25CF8[];
-/*
-static InitChainEntry sInitChain[] = {
-    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
+extern Gfx D_060003D0[];
+extern Gfx D_060031A0[];
+extern Gfx D_06003440[];
+extern AnimatedMaterial D_06001FF8;
+
+static ObjEndingGraphics sGraphics[] = {
+    { D_06003440, D_060031A0, NULL },
+    { NULL, D_060003D0, &D_06001FF8 },
 };
-*/
 
-extern unkStruct D_80C25CE0[];
-/*
-static unkStruct D_80C25CE0[2] = {
-    {0x06003440, 0x060031A0, 0x00000000},
-    {0x00000000, 0x060003D0, 0x06001FF8}
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
-*/
 
 void ObjEnding_Init(Actor* thisx, GlobalContext* globalCtx) {
     ObjEnding* this = THIS;
     AnimatedMaterial* texture;
 
-    Actor_ProcessInitChain(&this->actor, D_80C25CF8);
-    this->unk144 = &D_80C25CE0[this->actor.params];
-    if (false) {}
-    texture = this->unk144->texture;
-
+    Actor_ProcessInitChain(&this->actor, sInitChain);
```

Comment:
Normally if `thisx` is required, we use it everywhere possible in the function if it matches.